test/testdata/bootstrappolicy/bootstrap_cluster_roles.yaml

```diff
@@ -1710,16 +1726,24 @@ items:
     verbs:
     - get
   - apiGroups:
-    - project.openshift.io
     - ""
+    - project.openshift.io
     attributeRestrictions: null
     resources:
     - projectrequests
     verbs:
     - list
   - apiGroups:
-    - authorization.openshift.io
     - ""
+    - authorization.openshift.io
+    attributeRestrictions: null
+    resources:
+    - clusterroles
+    verbs:
+    - get
+    - list
```

Comment:
> This file is autogenerated by the test, what are you suggesting ?

the autogeneration is based on the permissions granted in the bootstrap policy.  It's odd to have a policy that allows get and list, but not watch.